cds-snc/covid-alert-app
# [WIP] Dynamically add debug plist properties

# Summary

Dynamically adds ATS exceptions to Info.plist for debug builds to enable live reload with the react-native metro server, and the ability to connect to a test diagnosis server without https. Negates the need for this: https://github.com/cds-snc/covid-alert-app#ios-local-development

# Test instructions

- Run debug build (either using the > button or running a test build) and you shouldn't see errors about a missing bundle URL.
- Run a local test diagnosis server and configure the app to connect to it.

# Reviewer checklist

This is a suggested checklist of questions reviewers might ask during their review:
